### ISSUE TITLE
 In a long video, only a few position values ​​are displayed and the rest are not displayed.

### ISSUE THREAD

**Issue by snbspd7** [latenitefilms/Metaburner#72]
There seems to be a problem in Metaburner where the position value is not displayed when the duration is 100 minutes.

I wonder if there is a solution.

**Comment by latenitefilms** [comment]
Apologies for the delayed response.

Are you able to share a FCPXML to reproduce this?

Most people using Metaburner are using it on feature length timelines - and we haven't had any issues so far about length causing problems.

**Comment by snbspd7** [comment]
Could it be that the XML I sent wasn't delivered?

**Comment by snbspd7** [comment]
[Sisterhood_MAIN_EDIT.fcpxmld.zip](https://github.com/user-attachments/files/18271416/Sisterhood_MAIN_EDIT.fcpxmld.zip)



If it doesn't go, I'll send it back to you.

**Comment by snbspd7** [comment]
Happy new year! You haven't found the cause yet, right? I am so curious so I am leaving a message.

**Comment by latenitefilms** [comment]
Thanks for chasing up @snbspd7!

I'm actually having trouble getting Metaburner to process your FCPXML at all - it's just crashing when attempting to process.

I'm still trying to work out exactly where and why it's failing.

There's a few weird things in the FCPXML when importing into a fresh Library on my side. For example, if you search for `34-14-T01` in the **Timeline Index**, there seems to be a clip that doesn't seem visible in the timeline. I can see the same thing with one of the `45-1-T02`, `62-6-T02` and `60-2-T05` instances. Do you see the same weirdness on your side?

Also, if you go into the **Clips** section of the **Timeline Index**, and sort by **Position**, there's quite a few clips that have a negative position.

I'll keep digging and see what I can uncover.

**Comment by latenitefilms** [comment]
After deleting the following "ghost clips" in the Final Cut Pro Timeline Index:

- 23-5-T02
- 34-14-T01
- 35-4-T05
- 45-1-T02
- 60-2-T05
- 62-6-T02
- 67-1-T01
- 75-2-T05
- 75-3-T02
- 75-4-T01
- 77-8-T01
- 78-4-T04
- 79-1-T02
- 79-2-T02 
- 79-4-T01
- 79-6-T03
- 80-20-T02

...everything seemed to work as intended with Metaburner.

So I think in this case, SOMETHING has happened to the Final Cut Pro project, that's corrupted/confused things.

If you check the above clips in the Final Cut Pro Timeline Index, can you also see these "ghost clips"? Does removing them fix the issue?

To be honest, I'm not really sure what the best solution to fix this is - as it SEEMS like it's a bug in Final Cut Pro.

**Comment by snbspd7** [comment]
Thanks for your reply.
I will test it and get back to you.
-----Original Message-----

**Comment by snbspd7** [comment]
Thank you for sharing the update. If the same issue persists even after deleting the ghost clips, it does seem likely that the problem originates within Final Cut Pro. As you mentioned, proceeding with the additional methods provided earlier is a good idea. Here are some further tips that might help:

**Comment by latenitefilms** [comment]
I think maybe your last message was incomplete?

**Comment by snbspd7** [comment]
Thank you for conducting such a thorough review. After deleting the ghost clips and running the test again, the same issue persists. It seems likely that the error is originating from Final Cut Pro. I will proceed with the additional methods you suggested.

**Comment by latenitefilms** [comment]
Ummm, strange - it's working fine here as far as I can tell after removing those ghost clips:

<img width="2051" alt="image" src="https://github.com/user-attachments/assets/5c5deb8b-e4b4-4d7e-a29b-95895b3ec263" />

What problems are you seeing exactly?

**Comment by snbspd7** [comment]
Yes, all the information is displayed correctly, but the issue is that the position values are only shown for a few clips, while the rest of the clips do not display any values.

**Comment by snbspd7** [comment]
Even when testing with a portion of the project, the position values are not displayed as shown in the image above. I wanted to resolve this issue.
<img width="3199" alt="스크린샷 2025-01-09 오전 10 11 05" src="https://github.com/user-attachments/assets/afca56f4-2644-486a-b6fe-4540d67a5605" />
